recipes/excitingtools/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "excitingtools" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/excitingtools-{{ version }}.tar.gz
+  sha256: cd1f6cf62e7e4b6a9b1d1de2fd26e0d9e3e032f5082c02be536aef5be3ae3dd2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=35.0.2
+    - setuptools-scm >=3.5.0
+    - pip
+  run:
+    - python >=3.6
+    - wheel >=0.35.0
```

Comment:
I won't hold up for this since it matches upstream but I'm 99% sure you're not relying on wheel at run-time.

Reply:
Thanks, very appreciated. I've taken another look and both the `setuptools-scm` pinning and addition of `wheels` as a run time dependency were a result of auto-generating the meta yaml with [grayskull](https://github.com/conda/grayskull). Admittedly I should have noticed, but I wasn't looking too hard 😅